packages/components/src/components/input/input.tsx

```diff
@@ -125,8 +125,10 @@ const Input = React.forwardRef<HTMLInputElement & HTMLTextAreaElement, TInputPro
                             ref={ref}
                             data-testid={data_testId}
                             {...props}
-                            className={classNames('dc-input__field', field_className, {
+                            className={classNames('dc-input__field', className, {
```

Comment:
@suisin-deriv Will this effect all the inputs that uses field_classname? I see some Input in p2p and bot package passes field_className to Input component.

Reply:
Let me double check on it :pray:

Reply:
@amina-deriv found some issue while checking on it. Working on the issue now 🙏